spec/tictac_spec.rb

```diff
@@ -0,0 +1,32 @@
+require './lib/logic'
+
```

Comment:
[OPTIONAL] Kindly use `require_relative` instead of `require` for local files as per ruby convention.